src/components/_proxied-dot-io/common/preview-product-select/index.tsx

```diff
@@ -0,0 +1,309 @@
+/**
```

Comment:
This file was largely copy pasted from the UI assembly branch. I adjusted the rendering to not be grouped and added some logic to set the cookie and reload on selection. It's also fixed in the corner now.